### PR TITLE
Epistola collegii module (#2)

### PR DESCRIPTION
**Pull request by kdmnk**

### CODE REVIEW

**Comment by kdmnk** [comment]
Seems fine now, sorry

File: app/Http/Controllers/StudentsCouncil/EpistolaController.php [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            $epistola = EpistolaNews::create($values);
+        }
+        return redirect(route('epistola'))->with('message', $updated ? __('general.successful_modification') : __('general.successfully_added'));
+    }
+
+    public function preview()
+    {
+        $this->authorize('send', EpistolaNews::class);
+
+        return (new EpistolaCollegii(EpistolaNews::where('sent', false)->get()->sortBy(function($result) {
```
Comment (luksan47) [inline app/Http/Controllers/StudentsCouncil/EpistolaController.php:120]: I would move the inner part to a variable

**Review by luksan47 — APPROVED** [review]
LGTM with some comments

File: app/Http/Controllers/StudentsCouncil/EpistolaController.php [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                if ($request->picture_upload != null && $request->picture_path != null)
+                    $fail(__('validation.upload_with_link'));
+            }]
+        ]);
+        $validator->validate();
+
+        //store and resize uploaded picture
+        if ($request->picture_upload != null) {
+            $path = $request->file('picture_upload')->store('', 'epistola');
+            Image::make(public_path('/img/epistola/' . $path))->resize(600, null, function ($constraint) {
```
[the remark this answers is not in the document]
Comment (kdmnk) [reply app/Http/Controllers/StudentsCouncil/EpistolaController.php:96]: What size would be optimal? These will be seen in little cards and in mails only.

**Pull request merged**